.github/workflows/build-and-test.yml

```
@@ -2,8 +2,7 @@ name: Build and Test
 on:
   push:
     branches:
-      - master
-      - develop
+      - '*'
   pull_request:
     paths:
       - '**'
```

```suggestion
      - '*'
```
I don't think we need two `*` here. I'll update this.